lib/show/show.sh

```diff
@@ -192,7 +197,7 @@ bpkg_show () {
         IFS=$'\n'
         return 0
       fi
-    done < "${BPKG_REMOTE_INDEX_FILE}"
+    done < "$BPKG_REMOTE_INDEX_FILE"
```

Comment:
I noticed this in more place and was curious: What is the rational for (not) having variables in curly brackets?

I.e. `${BPKG_REMOTE_INDEX_FILE}"` vs. `$BPKG_REMOTE_INDEX_FILE"`

I've grown accustomed to _with_ brackets (as that was the standard in the last company where I did BASH-heavy work) but I couldn't really give you a reason either for or against...

Reply:
I guess after sometime I became used to assuming some kind of special variable access (array, slice, replace, default) when I see `${...}` compared to just a value access for a variable with `$...` 🤷🏼 I am happy to get to a standard on this!

Reply:
Ages ago I used http://sideeffect.kr/popularconvention/ (which uses [the GitHub Dataset on google BigQuery ](https://codelabs.developers.google.com/codelabs/bigquery-github#0)) to see what dominant conventions were.

Maybe we can doe something similar for (the top BASH/Shell repos on GitHub](https://github.com/topics/shell?l=shell&o=desc&s=stars).

There are other areas where I think it would be nice to have some insight into which convention is more standard.